.github/workflows/java-api-client.yaml

```diff
@@ -0,0 +1,46 @@
+name: java-api-client
+
+on:
+  release:
+    types: [published]
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: Tag/version to publish
+
+jobs:
+  pypi:
```

Comment:
Can you rename the job to comply with the java client? maybe to "maven"

Reply:
Thanks!  Picked "`deploy`", because that's what it is... and should be understandable.  Preferred to avoid "maven" which is an implementation detail.